nucleus/deployment/common/src/main/java/org/glassfish/deployment/common/SecurityRoleMapper.java

```diff
@@ -37,16 +37,18 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
+// Portions Copyright [2024] Payara Foundation and/or affiliates
```

Comment:
Code in this file is copied from Eclipse GlassFish. Copyright should be attributed to "Contributors to the Eclipse Foundation" or OmniFish. The copied code is licensed under EPL or GPL, it cannot be distributed under the CDDL license.